### ISSUE TITLE
describer indices reset based on dragEnd

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#98]
The current dragEnd callback is called whenever a key is updated, not when the whole drag is done (all keys up). Both the increasing and decreasing describers reset their "levels" of alerts on dragEnd. For example, for increasing, level one is "jiggling more, wamer" and the next level is "jiggling faster, now hotter." And there is a similar progression for decreasing. 

I think it would be best for that to restart for every new drag. I'm not positive, but https://github.com/phetsims/scenery/issues/849 may fix this for us completely. No matter I still wanted to have a Friction specific issue to track how that scenery one effects this sim, and if there needs to be more tweaking. Right now the main way that this manifests is that is harder to get to the higher "leveled" alerts. Most of the time it just repeats the first level a lot.

**Comment by zepumph** [comment]
in #90 @terracoda mentioned that she was having a hard time hearing the "very hot" alert, that should be happening when you rub more even when atoms are breaking away. I think it may be nice to hard code that in to be separate from the current increasing array strategy in `TemperatureIncreasingDescriber`. So if you are alerting at a high temp, there won't be the sliding scale of alerts from that type, it will always be the "very hot" alert. I don't want to implement this until next week when @terracoda and I can have a longer feedback session on what is currently implemented.

**Comment by zepumph** [comment]
This issue is no longer needed as it will be fixed in #101

**Comment by zepumph** [comment]
this is actually different from #101. I'm going to reopen because the alerts other than "very hot" need work here.

After talking with @terracoda, we think the best way to proceed is to create an amount of time (that's right another threshold) from an end drag to another start, which if the time was small enough, we will just count it as the "same drag", such that the array of hotter alert indices don't start over. 

Basically right now we are restarting from the "coldest" relative alert (the first alert is "warmer") whenever a drag ends, but we will now change it such that if drags occur quick enough together, then they are counted as a single "drag session" where the alerts will progress throughout the whole session.

**Comment by zepumph** [comment]
Values can be tweaked here and there as part of #106, and "Very Hot" alert issues will be dealt with in #101, but this is sounding much better to me. I'm going to close.
